vignettes/SubDailyTimeSteps.Rnw

```diff
@@ -141,11 +146,14 @@ Load estimation can easily be done using unit-value data, rather than the user a
 
 <<echo=TRUE>>=
 # Get the data for May, 2013
-BrushyQ <- readNWISuv(Brushy, "00060", startDate="2013-05-01", 
+BrushyQ <- readNWISuv(Brushy, "00060", startDate="2013-05-01",
   endDate="2013-05-31", tz="America/New_York")
 # Rename the Flow column, must be done manually as renameNWISColumns
 # appends _Inst
-names(BrushyQ)[5] <- "Flow"
+flowColumn <- seq(4, 6, 1)[
+                which.min(lapply(BrushyQ[,4:6],
+                  function(x) sum(is.na(as.numeric(x)))))]
+names(BrushyQ)[flowColumn] <- "Flow"
```

Comment:
@wdwatkins, do you know the answer off the top of your head? i'm sorta wondering if `renameNWISColumns` followed by `gsub` to remove `'_Inst'` would be simpler